sqlmesh/core/console.py

```diff
@@ -756,7 +822,7 @@ def update_promotion_progress(self, snapshot: SnapshotInfoLike, promoted: bool)
             if self.verbosity >= Verbosity.VERBOSE:
                 action_str = "[green]promoted[/green]" if promoted else "[yellow]demoted[/yellow]"
                 self.promotion_progress.live.console.print(
-                    f"{snapshot.display_name(self.environment_naming_info, self.default_catalog, dialect=self.dialect)} {action_str}"
+                    f"{snapshot.display_name(self.environment_naming_info, self.default_catalog if self.verbosity < Verbosity.VERY_VERBOSE else None, dialect=self.dialect).ljust(self.PROGRESS_BAR_COLUMN_WIDTHS['name'])} {action_str}"
```

Comment:
Is there a way to left align the promoted/demoted column?

![image](https://github.com/user-attachments/assets/7e538bec-e05a-4cc4-a581-459da29ea609)